src/utils/typeHelpers/reducers/README.md

```diff
@@ -0,0 +1,73 @@
+# How to define reducers
+
+Reducers are functions that take the current state and an action,
+and return a new state. Reducers are pure functions, which means they
+should not mutate the state, but return a new state object
+
+In this project we recommend using minimal external dependencies,
+so we recommend using built in hooks like useState and useReducer
+
+If the component is simple, you can use useState, but if the component is
+complex, you can move to useReducer and move sideeffects into customHooks
```

Comment:
nit; how about this?

```suggestion
You can use useState if the component is simple, otherwise you can move to useReducer and move side effects into customHooks
```